Ginger/GingerCoreNET/ALMLib/Generic/ALMCore.cs

```diff
@@ -298,5 +299,10 @@ public static GingerCoreNET.ALMLib.ALMConfig GetDefaultAlmConfig()
             }
             return AlmConfig;
         }
+        public virtual void SetConnectValidationStatus(bool isConnect)
```

Comment:
No need the method, can set AlmCore.isConnectValidationDone =  true directly

Reply:
removed